pulsar-broker/src/main/java/org/apache/pulsar/broker/service/BrokerService.java

```diff
@@ -1005,14 +1009,14 @@ public CompletableFuture<Optional<Topic>> getTopic(final String topic, boolean c
                 });
             }
         } catch (IllegalArgumentException e) {
-            log.warn("[{}] Illegalargument exception when loading topic", topic, e);
+            log.warn("[{}] Illegalargument exception when loading topic", topicName.toString(), e);
```

Comment:
It looks like we don't need to call `toString` here.

Reply:
fixed.